agent/bench-scripts/postprocess/dbench-postprocess

```diff
@@ -1,135 +0,0 @@
-#!/usr/bin/perl
```

Comment:
Does deleting this file require a change to `agent/Makefile` (e.g., line 91)?

Reply:
No good deed goes unpunished :-(

Reply:
... and I had to do this, otherwise my original problem would come back to bite me.